### DISCUSSION TITLE
Genes taint on disconnect

### DISCUSSION THREAD

**Discussion by Mixmaster49** [FakeFishGames/Barotrauma#14933] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When a player disconnected from my server they instantly became brain dead and their genes tainted. If the game were working correctly, the genes should not taint on disconnect or brain death so that you can disconnect safely. 

### Reproduction steps

1. Start a multiplayer game.
2. Have another player with a gene splicer installed.
3. Have that player disconnect and become brain dead
4. Their genes should be tainted

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Addressed in the latest hotfix
